src/src/events/events.controller.ts

```diff
@@ -39,11 +39,10 @@ export class EventsController {
 	@Get('/list')
 	@UseGuards(new AuthGuard())
 	getEventsList(
-		@Body() body: any,
 		@Req() header: Request,
 		@Res() response: Response,
 	) {
-		return this.eventsService.getEventsList(body, header, response);
```

Comment:
req body not needed for filtering events?

Reply:
body i have already removed